### DISCUSSION TITLE
Publish V to scoop

### DISCUSSION THREAD

**Discussion by Milo123459** [vlang/v#11775] [Ideas]
Public V to scoop! The windows PowerShell package manager, I couldn't figure out how to setup the path correctly, so adding scoop would be helpful.

**Comment by JalonSolov** [comment]
To set up the path correctly, open an Administrator shell, cd to the V directory, and type `./v symlink` - that's all there is to it.

While a 'scoop' package might be useful, it will require someone who knows scoop and how to set up packages for it.

**Comment by Milo123459** [comment]
Makes sense, thanks, and here is how you set up a package: https://github.com/lukesampson/scoop/wiki/Creating-an-app-manifest

**Comment by andrewboldi** [comment]
Sire, you must translate make.bat to powershell if you want to use V with scoop.
